src/Administration/Resources/app/administration/src/app/component/rule/condition-type/sw-condition-generic/sw-condition-generic.spec.js

```diff
@@ -118,7 +113,7 @@ describe('components/rule/condition-type/sw-condition-generic', () => {
         Shopware.Store.get('ruleConditionsConfig').config = ruleConditionsConfig;
     });
 
-    it('should render fields and set condition values on change', async () => {
+    it.skip('should render fields and set condition values on change', async () => {
```

Comment:
Why are all of these skipped now?

Reply:
I talked with Segge about that.. it belongs to a pr of the admin team.